src/frameserver/PointGreyCam.cpp

```diff
@@ -539,7 +541,7 @@ void PointGreyCam<T>::setupAsyncTrigger(int trigger_mode,
         trigger_mode != 1  ||
         trigger_mode != 13 ||
         trigger_mode != 14) {
-        throw (rte("Trigger mode is unsupported."));
+      //throw (rte("Trigger mode is unsupported."));
```

Comment:
Sorry I meant a few lines down. To be clear, the way it is written right now, with line 542 uncommented, the function throws error with any input except -1.